src/AppBundle/Entity/Repository/SemesterRepository.php

```diff
@@ -48,6 +48,24 @@ public function findCurrentSemester()
             ->getOneOrNullResult();
     }
 
+    /**
+     * @return Semester
+     * @throws NonUniqueResultException
+     * @throws \Doctrine\ORM\ORMException
```

Comment:
To make the code easier to read, you could use an import for `ORMException`.

```suggestion
     * @throws ORMException
```
(Remember to add the import)